src/GiraffeGenerator.Core/OpenApi.fs

```diff
@@ -75,20 +76,84 @@ type PrimTypeKind =
         | Some "boolean", _ -> Bool
         | Some "string", maybeStringFormat -> String(StringFormat.Parse maybeStringFormat)
         | _ -> failwithf "Unexpected type: %s and format: %A" typeKind format
+        
+    member s.GetDefaultLiteral (value: IOpenApiPrimitive) =
+        let fw t = sprintf "%s literal has been found for non-%s type" t t |> failwith
```

Comment:
there is `failwithf` for failing with formatting

Reply:
Done